api/kyverno/v1/policy_types.go

```diff
@@ -82,6 +82,11 @@ func (p *Policy) BackgroundProcessingEnabled() bool {
 	return p.Spec.BackgroundProcessingEnabled()
 }
 
+// IsNamespaced indicates if the policy is namespace scoped
+func (p *Policy) IsNamespaced() bool {
+	return false
```

Comment:
why false ?

Reply:
Ignore , saw the PR description that we are converting `Policy` to `ClusterPolicy`